packages/storybook/src/stories/grid-reorder.stories.tsx

```diff
@@ -23,3 +25,27 @@ export const ReorderSmallDataset = () => {
     </div>
   );
 };
+
+export const DisableReorderOnSomeColumn = () => {
```

Comment:
Shouldn't we have a demo for this feature, as https://www.ag-grid.com/angular-grid/column-moving/#lock-position?

---

**Off topic**. It's IMHO, why we shouldn't work storybook first, but docs first. In the past, I was collecting a list of issues that we're creating because we were working Storybook first.

- https://github.com/mui-org/material-ui-x/issues/98 source: missing SSR support
- https://github.com/mui-org/material-ui-x/issues/209 source: isolated environment
- keyboard even conflicts when mounting two data grid on the same page (I can't find the GitHub issue related to it). source: isolated environment
- https://github.com/mui-org/material-ui-x/issues/895 source: isolated environment
- https://github.com/mui-org/material-ui-x/issues/974 source: isolated environment.

Reply:
You're right, I'll add a demo